lazy_dataset/database.py

```diff
@@ -214,13 +214,14 @@ def data(self):
 
 
 class JsonDatabase(Database):
-    def __init__(self, json_path: [str, Path]):
+    def __init__(self, *json_path: [str, Path]):
         """
 
         Args:
-            json_path: path to database JSON
+            json_path: One or multiple paths to database JSONs
 
         """
+        assert len(json_path) > 0, 'At least one database JSON is required!'
         self._json_path = json_path
```

Comment:
Can you support `JsonDatabase('1.json', '2.json') and JsonDatabase(['1.json', '2.json'])`